packages/commonwealth/client/scripts/controllers/app/login.ts

```diff
@@ -567,11 +573,9 @@ export async function handleSocialLoginCallback({
 
     // if email is not set, set the SSO email as the default email
     // only if its a standalone account (no account linking)
-    if (!userStore.getState().email && ssoEmail && profiles?.length === 1) {
-      await updateEmail({ email: ssoEmail })
-        .then(onUpdateEmailSuccess)
-        .catch(() => onUpdateEmailError(false));
-    }
+    // TODO: @timolegros - can we make this update directly in the signIn command? Do we have access to the ssoEmail?
+    if (!userStore.getState().email && ssoEmail && profiles?.length === 1)
+      await updateEmail({ email: ssoEmail as string });
```

Comment:
@timolegros did you see this one?

Reply:
No, didn't review the client. Yes, we can do this in the signIn command.